File: 2787.py
# Python 3.4.3

def par(n):
  return n%2 == 0

def impar(n):
  return n%2 != 0

def main():
  qtd_linhas = int(input())
  qtd_colunas = int(input())

  if (par(qtd_linhas) and par(qtd_colunas)) or impar(qtd_linhas) and impar(qtd_colunas):
    print(1)
  else:
    print(0)

if __name__ == '__main__':
	main()

File: 2787.c
#include <stdio.h>

int par(int n) {
  return n % 2 == 0;
}

int impar(int n) {
  return n % 2 != 0;
}

int main() {
  int l, c;
  scanf("%d %d", &l, &c);

  if ((par(l) && par(c)) || (impar(l) && impar(c))) {
    printf("1\n");
  } else {
    printf("0\n");
  }

	return 0;
}